.ci/Makefile

```diff
@@ -37,10 +37,12 @@ env:
 
 # Create Docker image
 build-docker: validate-version
+	@echo "Obsoleted, use GitHub actions"
```

Comment:
I would like this PR to remove the sections of this Makefile and scripts (like push-docker.sh), if they will become unused. In a separate PR is fine if helpful/necessary.